R/build-home-authors.R

```diff
@@ -4,17 +4,17 @@ data_authors <- function(pkg = ".", roles = default_roles()) {
 
   all <- pkg %>%
     pkg_authors() %>%
-    purrr::map(author_list, author_info)
+    purrr::map(author_list, author_info, pkg = pkg)
```

Comment:
it's a bit cumbersome to have to pass pkg all the time (it's needed by `markdown()` for adding the class to external links), maybe some refactoring in the future should put the package URL in the package environment or something like that? (or do the URL tweaking at some other point further down the line)